### DISCUSSION TITLE
When developing, how can I make Tina reload modified .md content whenever it changes?

### DISCUSSION THREAD

**Discussion by lokimckay** [tinacms/tinacms#3519] [Help] [answered]
I am using Tina with a NextJS site

I understand that the preferred workflow is to use Tina's sidebar to update content, but I temporarily want to follow this process while developing and getting started:

1. Make changes to `content/blog/myPost.md` using my code editor
1. Refresh my browser if needed (hopefully not required)
1. See changes in NextJS site

However I have observed that even the GraphQL API at `http://localhost:4001/altair/` doesn't contain the updated content when I re-execute a query to fetch the post. I think this means that my issue is not a result of NextJS caching the response using SSG or similar but rather with Tina itself.

Does Tina have some kind of cache that I can disable in development?

**Comment by logan-anderson** [comment, marked as answer]
@lokimckay You _should_ be able to add `--watchFolders content/blog/**/*` and that _should_ do what you are looking for. Let me know if you run into any issues.

**Reply by lokimckay** [reply]
Works perfectly 👌 thank you!

A note for others:
A hard refresh of the browser window is still required because NextJS is not aware that anything has changed
If you want NextJS to hot-reload you will have to consider something like [https://github.com/hashicorp/next-remote-watch](https://github.com/hashicorp/next-remote-watch)
